PyMC test helper function missing

When looking into https://github.com/pymc-labs/pymc-marketing/pull/186 I saw that a job is failing because:

```python
_______________ ERROR collecting tests/clv/test_distributions.py _______________
ImportError while importing test module '/home/runner/work/pymc-marketing/pymc-marketing/tests/clv/test_distributions.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/clv/test_distributions.py:7: in <module>
    from pymc.tests.helpers import select_by_precision
E   ModuleNotFoundError: No module named 'pymc.tests'
```

Can this be related to https://github.com/pymc-devs/pymc/pull/6540 @ricardoV94 ?

--- a/tests/clv/test_distributions.py
+++ b/tests/clv/test_distributions.py
@@ -4,7 +4,6 @@ import pytest
 from lifetimes import ParetoNBDFitter as PF
 from numpy.testing import assert_almost_equal
 from pymc import Model
-from pymc.tests.helpers import select_by_precision
 
 from pymc_marketing.clv.distributions import ContContract, ContNonContract, ParetoNBD
 
@@ -48,7 +47,7 @@ class TestContNonContract:
         assert_almost_equal(
             pm.logp(cnc, value).eval(),
             logp,
-            decimal=select_by_precision(float64=6, float32=2),
+            decimal=6,
             err_msg=str(pt),
         )
 
@@ -121,7 +120,7 @@ class TestContContract:
         assert_almost_equal(
             pm.logp(cc, value).eval(),
             logp,
-            decimal=select_by_precision(float64=6, float32=2),
+            decimal=6,
             err_msg=str(pt),
         )
 
@@ -217,7 +216,7 @@ class TestParetoNBD:
         assert_almost_equal(
             pm.logp(pareto_nbd, value).eval(),
             vectorized_logp(r, alpha, s, beta, value[..., 1], value[..., 0], T),
-            decimal=select_by_precision(float64=6, float32=2),
+            decimal=6,
             err_msg=str(pt),
         )
 

